c3/system/chip.py

```diff
@@ -3,11 +3,14 @@
 import numpy as np
 import tensorflow as tf
 
-from c3.c3objs import C3obj
+from c3.c3objs import C3obj, Quantity
 from c3.libraries.constants import kb, hbar
 from c3.libraries.hamiltonians import hamiltonians
 from c3.utils.qt_utils import hilbert_space_kron as hskron
-
+from scipy.optimize import fmin
+import tensorflow_probability as tfp
+from typing import List, Dict, Union
+import scipy.optimize
```

Comment:
Multiple imports of `scipy.optimize` in this block, possibly rendering the previous `from scipy.optimize import fmin` redundant.